metering/metering-installing-metering.adoc

```diff
@@ -7,7 +7,7 @@ toc::[]
 
 Review the following sections before installing metering into your cluster.
 
-To get started installing metering, first install the Metering Operator from OperatorHub. Next, configure your instance of metering by creating a `CustomResource`, referred to here as your MeteringConfig. Installing the Metering Operator creates a default MeteringConfig that you can modify using the examples in the documentation. After creating your MeteringConfig, install the metering stack. Last, verify your installation.
+To get started installing metering, first install the Metering Operator from OperatorHub. Next, configure your instance of metering by creating a `MeteringConfig` custom resource (CR). Installing the Metering Operator creates a default `MeteringConfig` resource that you can modify using the examples in the documentation. After creating your `MeteringConfig` resource, install the metering stack. Lastly, verify your installation.
```

Comment:
I can't find evidence of this, but at least in college it was _Last_ rather than _Lastly_.